src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.Process.cs

```diff
@@ -49,13 +50,28 @@ private static RemoteInvokeHandle RemoteInvoke(MethodInfo method, string[] args,
             }
 
             // If we need the host (if it exists), use it, otherwise target the console app directly.
-            string testConsoleAppArgs = "\"" + a.FullName + "\" " + t.FullName + " " + method.Name + " " + string.Join(" ", args);
+            var testConsoleAppArgs = new StringBuilder();
+            testConsoleAppArgs.Append(ExtraParameter);
```

Comment:
>It does look like the Unix builds are failing with TargetParameterCountException. I don't really know why - do you have any ideas?

Yes - it means you unconvered a bug in the Unix version of System.Diagnostic.Process. Here's the the suspect code:

https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs#L450

and here:

https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs#L518

you can see that this tokenizer is trying to implement the canonical behavior but gets it wrong for empty strings.

I think it's best to treat this as a blocking product issue and resolve that before attempting to go forward with this.